app/frontend/src/features/profile/ProfileTagInput.tsx

```diff
@@ -0,0 +1,229 @@
+      fontSize: "16px",
+      justifyContent: "space-between",
+    },
+    tag: {
+      padding: "0 14px",
+      margin: theme.spacing(1, 0),
+      fontSize: theme.typography.fontSize,
+      display: "flex",
+      alignItems: "center",
+    },
+    tagLabel: {
+      marginLeft: theme.spacing(1),
+    },
+    header: {
+      fontSize: theme.typography.fontSize,
+      borderBottomWidth: 1,
+      borderBottomStyle: "solid",
+      borderBottomColor: theme.palette.divider,
+      padding: theme.spacing(1, 2),
+    },
+    inputBase: {
+      padding: theme.spacing(2),
+      width: "100%",
+      borderBottomWidth: 1,
+      borderBottomStyle: "solid",
+      borderBottomColor: theme.palette.divider,
+      "& input": {
+        borderRadius: theme.shape.borderRadius,
+        backgroundColor: theme.palette.common.white,
+        padding: theme.spacing(1),
+        transition: theme.transitions.create(["border-color", "box-shadow"]),
+        borderWidth: 1,
+        borderStyle: "solid",
+        borderColor: theme.palette.divider,
+        "&:focus": {
+          boxShadow: `${fade(theme.palette.primary.main, 0.25)} 0 0 0 0.2rem`,
+          borderColor: theme.palette.primary.main,
+        },
+      },
+    },
+    paper: {
+      boxShadow: "none",
+      margin: 0,
+    },
+    option: {
+      minHeight: "auto",
+      alignItems: "flex-start",
+      padding: theme.spacing(1),
+      '&[aria-selected="true"]': {
+        backgroundColor: "transparent",
+      },
+      '&[data-focus="true"]': {
+        backgroundColor: theme.palette.action.hover,
+      },
+    },
+    popperDisablePortal: {
+      position: "relative",
+    },
+    checkbox: {
+      padding: 0,
+      marginRight: theme.spacing(1),
+    },
+  })
+);
+
+interface ProfileTagInputProps {
+  onChange: (_: any, value: string[]) => void;
+  value: string[];
+  options: string[];
+  label: string;
+  id: string;
+}
+
+export default function ProfileTagInput({
+  onChange,
+  value,
+  options,
+  label,
+  id,
+}: ProfileTagInputProps) {
+  const classes = useStyles();
+
+  const [open, setOpen] = useState<boolean>(false);
+  const anchorEl = useRef<null | HTMLButtonElement>(null);
+  const [valueState, setValueState] = useState<string[]>(value);
+  const [pendingValue, setPendingValue] = useState<string[]>([]);
+
+  useEffect(() => {
+    setValueState(value);
+  }, [value]);
+
+  const handleClick = (event: React.MouseEvent<HTMLElement>) => {
+    setPendingValue(value);
+    setOpen(true);
+  };
+
+  const handleClose = (
+    event: React.ChangeEvent<{}>,
+    reason: AutocompleteCloseReason
+  ) => {
+    if (reason === "toggleInput") {
+      return;
+    }
+    onChange(null, pendingValue);
+    setOpen(false);
+  };
+
+  const handleRemove = (tag: string) => {
+    onChange(
+      null,
+      value.filter((v) => v !== tag)
+    );
+  };
+
+  const popperId = open ? id : undefined;
+
+  return (
+    <>
+      <ButtonBase
+        disableRipple
```

Comment:
Do you get a ripple without this line? I thought it's disabled in the theme.

Reply:
Nope, you're right.